yaml/water.yaml

```diff
@@ -215,7 +215,7 @@ filters:
   #     - {kind: ocean}
   #   table: shp
   - filter: {meta.source: shp}
-    min_zoom: 0
```

Comment:
Ooops, I think this might have been the polygon, not the ocean label....

Reply:
oh, shoot!

Reply:
ah: `meta.source: shp` is for the **openstreetmapdata.com** source.

Reply:
Looks like they are all nodes:

![image](https://user-images.githubusercontent.com/853051/50025210-946c3a00-ff99-11e8-8fba-7fd8039cef35.png)